### ISSUE TITLE
Make Help Center Text Selectable

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#6946]
Currently text in the help center cannot be selected. Which is unexpected. Need to specify `user-select: all;` on the text in the help center.

**Comment by csossi** [comment]
Verified in QA
